### ISSUE TITLE
APKtool unable to recompile the navigation graph resource file

### ISSUE THREAD

**Issue by manhnvacrux** [iBotPeaches/Apktool#2410]
### Information
1. **Apktool Version (`apktool -version`)** - 2.4.1
2. **Operating System (Mac, Linux, Windows)** - Windows
3. **APK From? (Playstore, ROM, Other)** - Playstore

### Stacktrace/Logcat

**W: invalid resource directory name: E:\\.\insomia\res navigation**                                                                        
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [C:\Temp\brut_util_Jar_7294463487722717903.tmp, p, --forced-package-id, 127, --min-sdk-version, 19, --target-sdk-version, 29, --version-code, 48, --version-name, 2.0.2.RC-GP-Free(48), --no-version-vectors, -F, C:\Temp\APKTOOL8919051627574386978.tmp, -e, C:\Temp\APKTOOL5694668113728218948.tmp, -0, arsc, -I, C:\Users\Administrator\AppData\Local\apktool\framework\1.apk, -S, E:\\.\insomia\res, -M, E:\\.\insomia\AndroidManifest.xml] 

### Steps to Reproduce
1. Decompile
2. Recompile
3. APKtool unable to recompile apk file using **navigation component**

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? Yes
2. If you are trying to install a modified apk, did you resign it? Yes
3. Are you using the latest apktool version? Yes

**Comment by giacomoferretti** [comment]
Duplicate of #1811 #2219 #2338 #2391 #2393

TL;DR use --use-aapt2 to fix this problem. You will get some warnings but at least it rebuilds the apk.

**Comment by manhnvacrux** [comment]
Fixed
